app/src/main/java/dev/ricsarabia/cryptochallenge/ui/MainViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package dev.ricsarabia.cryptochallenge.ui
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dev.ricsarabia.cryptochallenge.data.repos.BitsoRepo
+import dev.ricsarabia.cryptochallenge.domain.*
+import kotlinx.coroutines.launch
+
+class MainViewModel : ViewModel() {
+    private val repo: BitsoRepo = BitsoRepo()
+    val loading = MutableLiveData(false)
```

Comment:
You should use a public live data for this, and a private mutable live data to change the value :D, also it would be better to specify the data type in your  Live Datas :D like in this example
```suggestion
   private var _loading = MutableLiveData<Boolean>()
   val loading: LiveData<Boolean> get() =  _loading
  
```